### PR TITLE
Send SIGKILL rather than SIGTERM on rpcchainvm shutdown

### PR DESCRIPTION
**Pull request by StephenButtolph**
## Why this should be merged

1. Unifies logic across operating systems.
2. Fixes unexpected plugin shutdown when sending SIGTERM to the node's process group

## How this works

- Because we expect to have already sent the Shutdown request over the gRPC, the sub-process should already have cleaned up all of its resources.
- Since the process should already have effectively exited, we can just kill it directly; skipping the existing graceful shutdown flow.

## How this was tested

- [ ] CI
- [x] Local network (darwin)

### CODE REVIEW

File: vms/rpcchainvm/vm.go [hunk 1]
```diff
@@ -39,20 +39,15 @@ func Serve(ctx context.Context, vm block.ChainVM, opts ...grpcutils.ServerOption
 	server := newVMServer(vm, opts...)
 	go func(ctx context.Context) {
 		defer func() {
+			signal.Stop(signals)
 			server.GracefulStop()
```
Comment (patrick-ogrady) [inline vms/rpcchainvm/vm.go:43]: With this PR, we don't except this to be hit right?

Comment (hexfusion) [reply vms/rpcchainvm/vm.go:43]: that is the only downside of this approach is we don't allow the gRPC server to gracefully shutdown.

Comment (StephenButtolph) [reply vms/rpcchainvm/vm.go:43]: correct - this would only be hit if the context finishes (which isn't part of the expected shutdown flow)

**Review by ceyonur — COMMENTED** [review]
just some nits

resim

**Comment by hexfusion** [comment]
replaced by #1988

**Pull request closed without merging**